app/helpers/score_helper.rb

```diff
@@ -1,11 +1,12 @@
 module ScoreHelper
-  def score_class(score)
-    if score > 1
-      'label-success'
-    elsif score > 0.5
-      'label-warning'
+  def score_opacity(score)
+    case score
+    when 0..1
+      score / 2
+    when 1..10
+      score / 10 + 0.5
```

Comment:
I just thought it would be too hard to see the number for very low scores. I don't mind removing it - also if you have a better idea on how to handle this, I'm all ears :D Since I can't really know the limits of the score, it's quite hard to find a good way to show it.